test/OpenTelemetry.Extensions.Enrichment.Tests/EnrichmentExtensionsTests.cs

```diff
@@ -0,0 +1,164 @@
+// <copyright file="EnrichmentExtensionsTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace OpenTelemetry.Extensions.Enrichment.Tests;
+
+public sealed class EnrichmentExtensionsTests
+{
+    private const string SourceName = nameof(EnrichmentExtensionsTests);
+
+    [Fact]
+    public void TracerProviderBuilder_AddTraceEnricherT_RegistersEnricher()
```

Comment:
nit: the "_" separator is not used in this repo. Would be good to stick with similar naming style.

Reply:
Thanks. Found one file which does, though - [GenevaTraceExporterTests.cs](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/test/OpenTelemetry.Exporter.Geneva.Tests/GenevaTraceExporterTests.cs). Probably a leftover after migrating from private repos.

Reply:
Renamed tests here